extensions/giveaways/Howto.md

```diff
@@ -0,0 +1,60 @@
+# How to install, setup and working with the commands.
+
+### Installing the actions.
```

Comment:
Would put Import code here using markup.

Reply:
Ok no problem I'll get on that for you.